docs/configuration.md

```diff
@@ -105,6 +111,57 @@ You can add this at the end of your config file:
 httpProxy = 'http://username:password@proxyAddress:port'
 ```
 
-### Log file
+## Log file
 
 On XOA, the log file for XO-server is in `/var/log/syslog`. It contains all the server information returned and can be a real help when you have trouble.
+
+## Reverse proxy
```

Comment:
What about old files?

Example: `docs/reverse_proxy.md`.

Reply:
We keep them for a bit for retro-compatibility issues (if we have old links around, it will continue to work)

Reply:
I think it would be better to remove them and to handle this at the HTTP server level, by generating both the new and old docs, and use the old as a fallback.

Reply:
Sounds like more than we can achieve on short term regarding all other priorities.